runtime/src/main.rs

```diff
@@ -42,20 +42,18 @@ pub extern "C" fn entry_point() -> ! {
             // treat global exception as a fatal error
             match e {
                 CaliptraError::RUNTIME_GLOBAL_EXCEPTION => handle_fatal_error(e.into()),
-                _ => caliptra_common::report_handoff_error_and_halt(
-                    "Runtime can't load drivers",
```

Comment:
similarly I think `[rt]` prefix will make these messages easier to read.

Reply:
Done